src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -44,6 +44,8 @@
 import android.view.View;
```

Comment:
Please update the copyright year to 2018 and include yourself as @author

Reply:
done